### ISSUE TITLE
Module fails testing when attempting to install with zef

### ISSUE THREAD

**Issue by tbrowder** [pdf-raku/PDF-Tags-raku#4]
## Problem

Attempt to  install `PDF::Tags`:

~~~
$ zef install PDF::Tags
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/t/pdf-font-cidfont.t
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFontType2.rakumod (PDF::Font::CIDFontType2)
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFont.rakumod (PDF::Font::CIDFont)
[PDF::Class] [31m===[0mSORRY![31m===[0m
[PDF::Class] Type 'PDF::COS::ByteString' is not declared
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/FontDescriptor/CID.rakumod (PDF::FontDescriptor::CID):15
[PDF::Class] ------> [32m        has PDF::COS::ByteString[33m⏏[31m $.Panose is entry; # Ten bytes for the [0m
[PDF::Class] Malformed has
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/FontDescriptor/CID.rakumod (PDF::FontDescriptor::CID):15
[PDF::Class] ------> [32m        has PDF::COS::[33m⏏[31mByteString $.Panose is entry; # Ten byte[0m
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFont.rakumod (PDF::Font::CIDFont):10
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFontType2.rakumod (PDF::Font::CIDFontType2):2
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/t/pdf-font-cidfont.t:12
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/t/pdf-resources.t
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/Type0.rakumod (PDF::Font::Type0)
[PDF::Class] [31m===[0mSORRY![31m===[0m Error while compiling /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFont.rakumod (PDF::Font::CIDFont)
[PDF::Class] [31m===[0mSORRY![31m===[0m
[PDF::Class] Type 'PDF::COS::ByteString' is not declared
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/FontDescriptor/CID.rakumod (PDF::FontDescriptor::CID):15
[PDF::Class] ------> [32m        has PDF::COS::ByteString[33m⏏[31m $.Panose is entry; # Ten bytes for the [0m
[PDF::Class] Malformed has
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/FontDescriptor/CID.rakumod (PDF::FontDescriptor::CID):15
[PDF::Class] ------> [32m        has PDF::COS::[33m⏏[31mByteString $.Panose is entry; # Ten byte[0m
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/CIDFont.rakumod (PDF::Font::CIDFont):10
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/lib/PDF/Font/Type0.rakumod (PDF::Font::Type0):10
[PDF::Class] at /home/tbrowde/.zef/store/PDF-Class-0.4.6.tar.gz/PDF-Class-0.4.6/t/pdf-resources.t:12
===> Testing [FAIL]: PDF::Class:ver<0.4.6>:auth<github:pdf-raku>
~~~
## Environment

Debian Buster 10.

Raku -v
    Welcome to 𝐑𝐚𝐤𝐮𝐝𝐨™ v2020.10.
    Implementing the 𝐑𝐚𝐤𝐮™ programming language v6.d.
    Built on MoarVM version 2020.10.

**Comment by dwarring** [comment]
Accidentally closed.

**Comment by tbrowder** [comment]
Hi, David. Any idea what's happening? I almost want to blame "zef" because I'm seeing similar errors when using App::Mi6 just installed with the latest zef.

**Comment by dwarring** [comment]
Hi Tom.
I golfed it down to:
```
use PDF::FontDescriptor;
role PDF::FontDescriptor::CID
    does PDF::FontDescriptor {
my  PDF::COS::ByteString $panose;
}

```
I forgot to `use PDF::COS::ByteString`, but I think [this recent PDF commit](https://github.com/pdf-raku/PDF-raku/commit/c5d04125c98a2e109ce93f3f0e314bbfffb54ab0#diff-45b06c9691a2682cb50e98cdfdeb53fbad6afa04e991da39c2eec35ac40a0a92) added it, so it happened to compile but
on recent versions of PDF only.

I'm thinking you ran into a problem with PDF::Class 0.4.6 missing a 'use PDF::COS::ByteString' and a slightly older version of PDF that doesn't happen to load it by accident.

Something to watch out for in future. I've fixed this in PDF::Class 0.4.7 (uploaded to CPAN) by adding the missing `use PDF::COS::ByteString` statement.
